tools/codegen/gen.py

```diff
@@ -224,11 +214,7 @@ def func(f: NativeFunction) -> Optional[str]:
 
             args_exprs_str = ', '.join(map(lambda a: a.name, args))
 
-            # See Note [Byte-for-byte compatibility]
-            # (return void_func() is valid C++)
             return_kw = "    return "
-            if returns_type == "void":
-                return_kw = " "
```

Comment:
as per the note above, return statements in a function with `void` return type are valid